pkg/render/intrusion_detection.go

```diff
@@ -1708,6 +1728,12 @@ func (c *intrusionDetectionComponent) intrusionDetectionControllerAllowTigeraPol
 			Protocol:    &networkpolicy.TCPProtocol,
 			Destination: networkpolicy.ESGatewayEntityRule,
 		})
+		egressRules = append(egressRules, v3.Rule{
```

Comment:
We should make a backlog item to eventually remove the old ESGateway rule from here (and elsewhere). We probably still need it for now for upgrade purposes, but would be nice to remove it in a few releases.